taskoutput/evergreen_sender_test.go

```diff
@@ -179,33 +179,35 @@ func TestSend(t *testing.T) {
 	})
 	t.Run("FlushesAtInterval", func(t *testing.T) {
 		mock := newSenderTestMock(ctx)
-		mock.sender.opts.FlushInterval = 2 * time.Second
+		mock.sender.opts.FlushInterval = 4 * time.Second
 
-		m := message.ConvertToComposer(level.Debug, utility.RandomString())
-		mock.sender.Send(m)
-		require.NotEmpty(t, mock.sender.buffer)
+		data := utility.RandomString()
+		mock.sender.buffer = []log.LogLine{{Data: data}}
+		prevFlush := time.Now().Add(-mock.sender.opts.FlushInterval)
+		mock.sender.lastFlush = prevFlush
 		go mock.sender.timedFlush()
-		time.Sleep(2 * time.Second)
+		time.Sleep(3 * mock.sender.opts.FlushInterval / 2)
```

Comment:
what's behind this calculation?

Reply:
On second thought, I decided to simplify this. I realized I can achieve the same thing by rounding the difference between flush times to the nearest second :).